dashboard/test/config/initializers/mysql_avoid_utf8_alias_in_schema_dump_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class MysqlAvoidUtf8AliasInSchemaDumpTest < ActiveSupport::TestCase
```

Comment:
Some not happy test I would recommend
-Testing for bad versions 
- Testing for exceptions when requesting the version
- Default values if you decide to add this

```
test 'handles unexpected mysql version formats gracefully' do
  ActiveRecord::Base.connection.stubs(:select_value).returns('unexpected-format')
  assert_raises(RuntimeError) { @schema_dumper.send(:current_mysql_version) }
end

test 'uses fallback version on error fetching mysql version' do
  ActiveRecord::Base.connection.stubs(:select_value).raises('Simulated connection error')
  assert_equal(5.7, @schema_dumper.send(:current_mysql_version))
end

```